doc/source/user_guide/signal_tools.rst

```diff
@@ -143,3 +143,29 @@ The default operational mode is ``inplace=False`` (a new signal object is return
 
     >>> s.remove_negative(0.1)
 
+
```

Comment:
These functions do not operate on the signal, so I would like to keep them in the utils page of the documentation.

Reply:
(unit conversion and grating equation)